SingularityService/src/main/java/com/hubspot/singularity/scheduler/SingularityUsagePoller.java

```diff
@@ -86,7 +91,9 @@ public void runActionOnPoll() {
 
             double taskCpusUsed = ((usage.getCpuSeconds() - lastUsage.getCpuSeconds()) / (usage.getTimestamp() - lastUsage.getTimestamp()));
 
-            updateUsagesPerRequestType(usagesPerRequestType, getRequestType(taskUsage), usage.getMemoryRssBytes(), taskCpusUsed);
+            if (getRequestType(taskUsage).isLongRunning() ||  isConsideredLongRunning(taskUsage)) {
+              updateLongRunningTasksUsage(longRunningTasksUsage, usage.getMemoryRssBytes(), taskCpusUsed);
+            }
```

Comment:
Added this to also include the usage for tasks that are non long running, but run for a considerable amount of time to warrant adding their usage